spec/unit/providers/aws/credentials_spec.rb

```diff
@@ -1,45 +1,47 @@
 require 'chef/provisioning/fog_driver/providers/aws/credentials'
 
 describe Chef::Provisioning::FogDriver::Providers::AWS::Credentials do
-  describe "#load_ini" do
+  subject { Chef::Provisioning::FogDriver::Providers::AWS::Credentials.new }
```

Comment:
I used subject because it's consistent with the other specs in this project. I can change to let if you prefer. Let me know.